src/LightInject.Tests/AssemblyScannerTests.cs

```diff
@@ -298,6 +298,32 @@ public void Register_AssemblyWithFunc_CallsAssemblyScanner()
         }
 
         [Fact]
+        public void Register_Assembly_CallsAssemblyScannerWhenAssemblyScanningEnabled()
```

Comment:
I'm not familiar with the convention used to name the unit tests in this project so I will welcome suggestions if you can think of a better name ;)